src/NHSD.BuyingCatalogue.Solutions.Persistence/Repositories/PriceRepository.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Data.Infrastructure;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Persistence;
+using NHSD.BuyingCatalogue.Solutions.Persistence.Models;
+
+namespace NHSD.BuyingCatalogue.Solutions.Persistence.Repositories
+{
+    public sealed class PriceRepository : IPriceRepository
+    {
+        private readonly IDbConnector _dbConnector;
+
+        private const string ListCataloguePricesSql = @"
+        SELECT  cp.CatalogueItemId,
+                ci.[Name] as CatalogueItemName,
+                cp.CataloguePriceId,
+                cp.CataloguePriceTypeId,
+                pu.PricingUnitId,
+                pu.[Name] as PricingUnitName,
+                pu.[Description] as PricingUnitDescription,
+                pu.TierName as PricingUnitTierName,
+                cp.TimeUnitId,
+                cp.CurrencyCode,
+                cp.Price AS FlatPrice,
+                cptr.BandStart,
+                cptr.BandEnd,
+                cptr.Price AS TieredPrice
+        FROM    CataloguePrice cp
+                INNER JOIN PricingUnit pu ON pu.PricingUnitId = cp.PricingUnitId
+                INNER JOIN CatalogueItem ci ON ci.CatalogueItemId = cp.CatalogueItemId
+                LEFT OUTER JOIN CataloguePriceTier cptr ON cptr.CataloguePriceId = cp.CataloguePriceId
```

Comment:
Can you add the following two things?

1. `AS` for your table aliases? I think it improves readability (and I think we agreed to use it in standards).
2. Schema qualifiers for your table names, all `dbo` in this case – it just leaves no doubt as to intended schema (and saves SQL Server a little work too).

Reply:
👍